system/network/check-banner.rb

```diff
@@ -0,0 +1,128 @@
+---
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: check-banner
+  namespace: default
+spec:
+  command: check-banner.rb -q {{ .labels.check_banner_pattern | default banner }}
```

Comment:
I believe strings need to be in double quotes for token syntax. At least I ran into a parse error on some other checks I wrote.
```suggestion
  command: check-banner.rb -q {{ .labels.check_banner_pattern | default "banner" }}
```

Reply:
I'll test this and make changes as needed. Thanks for catching this!